.github/actions/e2e/cleanup/action.yaml

```diff
@@ -30,6 +30,18 @@ runs:
     - uses: ./.github/actions/e2e/install-eksctl
       with:
         eksctl_version: v0.147.0
+    - name: delete-security-group
+      shell: bash
+      run: | 
+        aws ec2 describe-security-groups \
```

Comment:
Can we describe why we delete the security group here. Either in the YAML or in the script as a comment?